src/renderers/WebGLRenderer.js

```diff
@@ -1093,6 +1093,9 @@ class WebGLRenderer {
 
 			const shadowsArray = currentRenderState.state.shadowsArray;
 
+			// muse be done before shadowMap.render, so that the number is the the same for shadows and real objects
+			this.info.render.frame ++;
```

Comment:
I think this was an important fix since it potentially produced frame late effects when rendering the shadow map (because `WebGLShadowMap.render()` relied on an outdated frame value).